README.md

```diff
@@ -9,7 +9,7 @@ This service depends on:
 # Dev setup
```

Comment:
Can you amend this whole section to reference the shared readmes please? So for dev setup, testing, pre-commit. But keep the note about adding stuff to `pyrpoject.toml` somewhere as that won't be in shared docs.

Reply:
added the testing read me section now so all of above are linked to corresponding shared readmes